src/components/pages/InstructorHome/InstructorCalender.js

```diff
@@ -1,10 +1,46 @@
-import React from 'react';
-import { Calendar } from 'antd';
+import React, { useState } from 'react';
+import { Calendar, Badge } from 'antd';
 import './index.css';
-const InstrcutorCalender = () => {
-  function onPanelChange(value, mode) {
-    console.log(value, mode);
+const InstructorCalender = ({ dummyData }) => {
+  const [courses, setCourses] = useState(dummyData);
+  const [course, setCourse] = useState();
+
+  function getListData(value) {
+    courses.map(list => {
+      setCourse(list);
+    });
+    let listData;
+    switch (value.date()) {
+      case 8:
```

Comment:
Glad you got the 'mapping' flowing correctly. Looks good. I wonder what we could do to get the calendar to read the month and date to differentiate between months.

Reply:
Currently, the dummy data isn't complete once we get a date the instructor picks we will be able to restrict the cases to that particular day. Should be an easy fix